src/tests/Signup/index.test.jsx

```diff
@@ -28,4 +28,60 @@ describe('Sign up Component', () => {
     wrapper.find('#firstName').simulate('change', event);
     expect(spy).toHaveBeenCalledTimes(1);
   });
+
+  it('should handle text lastname input', () => {
+    const event = {
```

Comment:
How is this test related to the ticket?

Reply:
Test coverage had dropped and when I checked where the drop was coming from it showed that the drop was coming from it showed the drop was arising from Sign up as shown in the screenshot below so I worked on raising the coverage
![image](https://user-images.githubusercontent.com/39259834/54535244-2e7bbe00-499f-11e9-827e-050bfad9c715.png)